poem_generator/generator.py

```diff
@@ -1,12 +1,15 @@
 from poem_generator.generators.next_line import mbart_fi_single_line
+from poem_generator.generators.first_line import mbart_fi_first_line
 from poem_generator.io.candidates import PoemLine, PoemLineList
 from poem_generator.io.config import PoemGeneratorConfiguration
 
 
 class PoemGenerator:
-    def __init__(self, config: PoemGeneratorConfiguration):
+    def __init__(self, config: PoemGeneratorConfiguration, keywords: str):
```

Comment:
To avoid re-loading the model every time a new poem starts, the `keywords` argument should not be passed to the constructor but to the method `PoemGenerator.get_first_line_candidates`.